deploy/helm/README.md

```diff
@@ -2,10 +2,111 @@
+- [Features](#features)
+  - [Metrics Collection](#metrics-collection)
+  - [Logs Collection](#logs-collection)
+  - [Events Collection](#events-collection)
+  - [Manifests Collection](#events-collection)
+  - [Auto Instrumentation (disabled by default)](#auto-instrumentation-disabled-by-default)
+  - [AWS Fargate (disabled by default)](#aws-fargate-disabled-by-default)
+  - [eBPF Network Observability](#ebpf-network-observability)
+  - [OpenShift Support (disabled by default)](#openshift-support-disabled-by-default)
+  - [Auto Update (disabled by default)](#auto-update-disabled-by-default)
+- [Receive 3rd party metrics](#receive-3rd-party-metrics)
+
+## Architecture
+
+```mermaid
+flowchart LR
+
+%% Kubernetes Control Plane
+subgraph Kubernetes
+    KubeAPI:::k8s
+    CoreDNS:::k8s
+    ETCD:::k8s
+
+    PrometheusWorkloads["Kubernetes Workloads"]:::deployment
+
+    subgraph "SWO K8s Collector Chart"
+        subgraph "Otel Operator Chart"
+            OtelOperator["OtelOperator"]:::deployment
+            TargetAllocator["TargetAllocator"]:::deployment
+        end
+        subgraph "Cert Manager Chart"
+            CertManager["CertManager"]:::deployment
+        end
+        subgraph "KubeStateMetrics Chart"
+            KubeStateMetrics["kube-state-metrics"]:::deployment
+        end
+
+        MetricsCollector["Metrics Collector"]:::deployment
+        DiscoveryCollector["Discovery Collector"]:::statefulset
+        EventCollector["Event Collector"]:::deployment
+        Reducer["eBPF Network - Reducer"]:::deployment
+        K8sCollector["eBPF Network - K8s Collector"]:::deployment
+
+        KernelCollector["eBPF Network - Kernel Collector"]:::daemonset
+        NodeCollector["Node Collector"]:::daemonset
+
+        OtelCollectorCrd["OpenTelemetryCollector CRD"]:::crd
+
+        OtelCollectorCrd---DiscoveryCollector
+        DiscoveryCollector--> |"get targets"| TargetAllocator
+    end
+
+    subgraph "Nodes" 
+        WorkloadLogs["Container Logs"]
+        eBPF["Workload Communication"]
+    end
+end
+
+
+%% SWO with explicit interface node
+subgraph "SolarWinds Observability (SWO)"
+    SWO_Interface["SWO OTLP"]:::swo
+end
+
+%% Connections explicitly via SWO Interface
+KernelCollector --> Reducer
+KernelCollector --> eBPF
+NodeCollector --> |"scrape"| PrometheusWorkloads
+DiscoveryCollector --> |"scrape"| PrometheusWorkloads
+OtelOperator---> |"autoinstruments"| PrometheusWorkloads
+NodeCollector --> WorkloadLogs
+NodeCollector --> |"ingest"| SWO_Interface
+
+Reducer --> K8sCollector
+K8sCollector --> |"watch"| KubeAPI
+TargetAllocator --> |"discover"| KubeAPI
+KubeStateMetrics --> KubeAPI
+Reducer --> |"ingest"| MetricsCollector
+
+MetricsCollector --> |"scrape"| KubeStateMetrics
+MetricsCollector --> |"scrape"| CoreDNS
+MetricsCollector --> |"scrape"| ETCD
+%% Trick to move SWO more to the right
+MetricsCollector --> X[" "]:::invisible --> |"ingest"| SWO_Interface
```

Comment:
Why is there the additional `X[" "]:::invisible` node?

Reply:
This is hack to move SWO_Interface more to the right, but I will remove this hack as it does not improve layout that much.